backend/app/api/diagnosis.py

```
@@ -60,41 +62,54 @@ async def create_diagnosis_history(
+            "type": "done",
+            "data": {
+                "id": new_history.id,
+                "pest": new_history.pest,
+                "crop": new_history.crop,
+                "region": new_history.region,
+                "analysis_result": new_history.analysis_result,
+                "image_url": new_history.image_url,
+                "created_at": new_history.created_at.isoformat()
+            }
+        }
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)